Conceptpower+Spring/src/main/java/edu/asu/conceptpower/servlet/rdf/RDFMessageFactory.java

```diff
@@ -0,0 +1,148 @@
+            Property rdfsSeeAlso = model.createProperty(rdf + "seeAlso");
+            Property skosRelated = model.createProperty(skos + "related");
+
+            Resource resource = model.createResource(creator.getURI(entry));
+
+            resource.addProperty(identifiers, entry.getId());
+            resource.addProperty(madsrdfProperty, entry.getWord());
+            resource.addProperty(skosProperty, entry.getWord());
+            resource.addProperty(schemaName, entry.getWord());
+            resource.addProperty(schemaDescription, entry.getDescription());
+
+            if (entry.getConceptList() != null) {
+                resource.addProperty(madsrdfCL, creator.getConceptListURI(entry));
+            }
+
+            if (entry.getCreatorId() != null) {
+                resource.addProperty(csCreator, entry.getCreatorId());
+            }
+
+            if (entry.getModified() != null) {
+                resource.addProperty(csCreator, entry.getCreatorId());
+            }
+
+            if (entry.getEqualTo() != null) {
+                String[] equals = entry.getEqualTo().split(",");
+                for (String equal : equals) {
+                    resource.addProperty(owlEqualTo, equal);
+                }
+            }
+
+            if (entry.getSimilarTo() != null) {
+                String[] similarTo = entry.getSimilarTo().split(",");
+                for (String similar : similarTo) {
+                    resource.addProperty(rdfsSeeAlso, similar);
+                }
+            }
+
+            if (entry.getWordnetId() != null) {
+                resource.addProperty(owlEqualTo, creator.getWordnetURI(entry));
+            }
+
+            if (entry.getSynonymIds() != null) {
+                String[] synonymIds = entry.getSynonymIds()
+                        .split(edu.asu.conceptpower.servlet.core.Constants.SYNONYM_SEPARATOR);
+                for (String synonymId : synonymIds) {
+                    resource.addProperty(skosRelated, synonymId);
+                }
+            }
+
+            Resource typeResource = model.createResource(skosType);
+            resource.addProperty(RDF.type, typeResource);
+
+            Resource typeRes = model.createResource(authorityType);
+            resource.addProperty(RDF.type, typeRes);
+
+            if (entry.getTypeId() != null) {
+                Resource typesRes = model.createResource(creator.getTypeURI(entry));
+                resource.addProperty(RDF.type, typesRes);
+            }
+
+            String matchesURI = entry.getTypeId();
+            if (matchesURI != null) {
+                String matches = typeManager.getType(matchesURI).getMatches();
+                if (matches != null && !matches.equalsIgnoreCase("")) {
+                    Resource cdocResource = model.createResource(matches);
+                    resource.addProperty(RDF.type, cdocResource);
+                }
+            }
+            model.write(out, syntax);
```

Comment:
Currently (wrong):
```
<rdf:RDF
    xmlns:madsrdf="http://www.loc.gov/mads/rdf/v1#"
    xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
    xmlns:dcterms="http://purl.org/dc/terms/"
    xmlns:cs="http://purl.org/vocab/changeset/schema#"
    xmlns:owl="http://www.w3.org/2002/07/owl#"
    xmlns:cidoccrm="http://www.cidoc-crm.org/cidoc-crm/"
    xmlns:skos="http://www.w3.org/2008/05/skos#"
    xmlns:schema="http://schema.org/">
  <madsrdf:Authority rdf:about="http://www.digitalhps.org/concepts/WID-02374451-N-01-horse">
    <dcterms:identifiers>WID-02374451-N-01-horse</dcterms:identifiers>
    <schema:name>horse</schema:name>
    <owl:sameAs>http://www.digitalhps.org/wordnet/WID-02374451-N-01-horse</owl:sameAs>
    <skos:related>WID-02374451-N-02-Equus_caballus</skos:related>
    <madsrdf:isMemberOfMADSCollection>http://www.digitalhps.org/lists/WordNet</madsrdf:isMemberOfMADSCollection>
    <schema:description>solid-hoofed herbivorous quadruped domesticated since prehistoric times</schema:description>
    <rdf:type rdf:resource="http://www.w3.org/2008/05/skos#Concept"/>
    <madsrdf:authoritativeLabel>horse</madsrdf:authoritativeLabel>
    <skos:prefLabel>horse</skos:prefLabel>
  </madsrdf:Authority>
</rdf:RDF>
<rdf:RDF
    xmlns:madsrdf="http://www.loc.gov/mads/rdf/v1#"
    xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
    xmlns:dcterms="http://purl.org/dc/terms/"
    xmlns:cs="http://purl.org/vocab/changeset/schema#"
    xmlns:owl="http://www.w3.org/2002/07/owl#"
    xmlns:cidoccrm="http://www.cidoc-crm.org/cidoc-crm/"
    xmlns:skos="http://www.w3.org/2008/05/skos#"
    xmlns:schema="http://schema.org/">
  <skos:Concept rdf:about="http://www.digitalhps.org/concepts/WID-03538037-N-01-horse">
    <dcterms:identifiers>WID-03538037-N-01-horse</dcterms:identifiers>
    <schema:description>a padded gymnastic apparatus on legs</schema:description>
    <madsrdf:authoritativeLabel>horse</madsrdf:authoritativeLabel>
    <owl:sameAs>http://www.digitalhps.org/wordnet/WID-03538037-N-01-horse</owl:sameAs>
    <skos:prefLabel>horse</skos:prefLabel>
    <schema:name>horse</schema:name>
    <madsrdf:isMemberOfMADSCollection>http://www.digitalhps.org/lists/WordNet</madsrdf:isMemberOfMADSCollection>
    <rdf:type rdf:resource="http://www.loc.gov/mads/rdf/v1#Authority"/>
    <skos:related>WID-03538037-N-02-gymnastic_horse</skos:related>
  </skos:Concept>
  <madsrdf:Authority rdf:about="http://www.digitalhps.org/concepts/WID-02374451-N-01-horse">
    <dcterms:identifiers>WID-02374451-N-01-horse</dcterms:identifiers>
    <schema:name>horse</schema:name>
    <owl:sameAs>http://www.digitalhps.org/wordnet/WID-02374451-N-01-horse</owl:sameAs>
    <skos:related>WID-02374451-N-02-Equus_caballus</skos:related>
    <madsrdf:isMemberOfMADSCollection>http://www.digitalhps.org/lists/WordNet</madsrdf:isMemberOfMADSCollection>
    <schema:description>solid-hoofed herbivorous quadruped domesticated since prehistoric times</schema:description>
    <rdf:type rdf:resource="http://www.w3.org/2008/05/skos#Concept"/>
    <madsrdf:authoritativeLabel>horse</madsrdf:authoritativeLabel>
    <skos:prefLabel>horse</skos:prefLabel>
  </madsrdf:Authority>
</rdf:RDF>
```